tests/TestCase/Validation/ValidationTest.php

```diff
@@ -1102,6 +1103,7 @@ public function testDateMmddyyyy()
         $this->assertFalse(Validation::date('11.31.2006', ['mdy']));
         $this->assertFalse(Validation::date('11/31/2006', ['mdy']));
         $this->assertFalse(Validation::date('11 31 2006', ['mdy']));
+        $this->assertFalse(Validation::date("12-27-2006\n", ['mdy']));
```

Comment:
That's right.
I understood that it is a task that should be handled separately from `Validation` class validation.